packages/pluggableWidgets/web-view-native/src/WebView.xml

```diff
@@ -24,6 +24,19 @@
                 <description/>
             </property>
         </propertyGroup>
+        <propertyGroup caption="Callback">
+            <property key="onMessage" type="action" required="false">
+                <caption>On message</caption>
+                <description>Used to communicate from the loaded webpage to the native app trough window.ReactNativeWebView.postMessage([string])</description>
+            </property>
+            <property key="onMessageInput" type="attribute" required="false">
+                <caption>message attribute</caption>
```

Comment:
Hello, one little change request here, first letter should be uppercase.

Reply:
Nice catch